libraries/cli/src/config.cpp

```diff
@@ -160,6 +160,22 @@ Config::Config(int argc, const char* argv[]) {
     Json::Value config_json = Tools::readJsonFromFile(config);
     Json::Value wallet_json = Tools::readJsonFromFile(wallet);
 
+    auto override = [&]() {
```

Comment:
override is C++ keyword(identifier with special meaning), add some prefix or adjust somehow the name for this function